common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -252,7 +252,7 @@ object ReutersParser extends ImageProcessor {
   def apply(image: Image): Image = image.metadata.credit match {
     // Reuters and other misspellings
     // TODO: use case-insensitive matching instead once credit is no longer indexed as case-sensitive
-    case Some("REUTERS") | Some("Reuters") | Some("RETUERS") | Some("REUTERS/") | Some("via REUTERS") | Some("VIA REUTERS") => image.copy(
+    case Some("REUTERS") | Some("Reuters") | Some("RETUERS") | Some("REUETRS") | Some("REUTERS/") | Some("via REUTERS") | Some("VIA REUTERS") | Some("via Reuters") => image.copy(
```

Comment:
Do we catch all of these permutations, or do some slip through? We could use a fuzzy match to detect spelling errors in this and other fields if it was worth the effort ... (in another PR 😁)

Reply:
Good point, but this whole business logic will change as soon as we review and merge BBC PRs, so I would just make those able to deal with that.

Reply:
Maybe we should do that instead?

Reply:
> we should do that

Do what? Review and merge BBC PRs? Oh, yes! (but not instead, we can adjust supplierProcessor logic after). We shoud also engineer BBC PRs such, that all manner of crazy stuff currently there and stuff we would like to do, but can’t now, should be possible. Eg for here, as noted, matching a supplier on last token would help: we are loosing some pics from free tier because of that.